dsc/workflows/base/__init__.py

```diff
@@ -315,6 +317,85 @@ def finalize_items(self) -> None:
 
         Must NOT be overridden by workflow subclasses.
         """
+        logger.info(
+            f"Processing DSS result messages from the output queue '{self.output_queue}'"
+        )
+        sqs_results_summary = {
+            "received_messages": 0,
+            "ingest_success": 0,
+            "ingest_failed": 0,
+            "ingest_unknown": 0,
+        }
+
+        # retrieve and create map of result messages
+        sqs_client = SQSClient(
+            region=CONFIG.aws_region_name, queue_name=self.output_queue
+        )
+        logger.info(
+            f"Processing DSS result messages from the output queue '{self.output_queue}'"
+        )
+        result_message_map = {}
+        for message in sqs_client.receive():
+            try:
+                self.validate_result_message(message)
+                item_identifier = message["MessageAttributes"]["PackageID"]["StringValue"]
+                result_message_map[item_identifier] = message
+            except InvalidSQSMessageError:
+                logger.exception(f"Failure parsing message '{message["MessageId"]}'")
+                continue
```

Comment:
Given the updated flow, invalid messages are not added to `result_message_map` which effectively triggers the same `INGEST_UNKNOWN` status via lines 363-368 rather in the previous `except` block for an individual message

Reply:
Thinking about this more, the exception will no longer be recorded in the `status_details` attribute which may or may not be a problem. But I do think invalid result messages will be exceedingly rare, I *believe* DSS and SQS will bounce them well before this point, but we should still have checks in case they don't